Doc/whatsnew/3.11.rst

```diff
@@ -230,26 +236,30 @@ See :pep:`646` for more details.
 Serhiy Storchaka and Jelle Zijlstra. PEP written by Mark Mendoza, Matthew
 Rahtz, Pradeep Kumar Srinivasan, and Vincent Siles.)
 
+
+.. _whatsnew311-pep655:
+
 PEP 655: Marking individual ``TypedDict`` items as required or not-required
 ---------------------------------------------------------------------------
 
 :data:`~typing.Required` and :data:`~typing.NotRequired` provide a
 straightforward way to mark whether individual items in a
-:data:`~typing.TypedDict` must be present. Previously this was only possible
+:class:`~typing.TypedDict` must be present. Previously, this was only possible
 using inheritance.
 
-Fields are still required by default, unless the ``total=False``
-parameter is set.
-For example, the following specifies a dictionary with one required and
-one not-required key::
+All fields are still required by default,
+unless the *total* parameter is set to ``False``,
+in which case all fields are still not-required by default.
+For example, the following specifies a :class:`!TypedDict`
+with one required and one not-required key::
 
    class Movie(TypedDict):
       title: str
       year: NotRequired[int]
 
-   m1: Movie = {"title": "Black Panther", "year": 2018}  # ok
-   m2: Movie = {"title": "Star Wars"}  # ok (year is not required)
-   m3: Movie = {"year": 2022}  # error (missing required field title)
+   m1: Movie = {"title": "Black Panther", "year": 2018}  # OK
+   m2: Movie = {"title": "Star Wars"}  # OK (year is not required)
+   m3: Movie = {"year": 2022}  # ERROR (missing required field title)
```

Comment:
What's the motivation for the ALLCAPS change here? It looks like shouting to me.

Reply:
It is intended to draw emphasis to the key contrast these examples are included to illustrate—the first and second are valid, while the third is invalid and produces an error. Also, "OK" is of course the correct, canonical capitalization of "OK", while it is "Error" is ALLCAPSed given that in

Reply:
"ERROR" still seems ugly and unnecessary to me; I don't really see the need for a change here.

Reply:
It also goes against the style in the `typing` docs — there are several `# error` comments,  but none of them are ALLCAPS: https://docs.python.org/3/library/typing.html

Reply:
Personally, I value clarity and explicitness over subjective aesthetic preference, given a stated rationale. Also, I'm not sure how relevant the conventions of the `typing` stdlib doc are here, given it is a separate document from this one, and those particular choices may or may not have a clear, thought-out rationale.

But what do others think? @ezio-melotti @JelleZijlstra ?

If an all-caps `ERROR` is simply too objectionable, we can at least give it proper case

```suggestion
   m3: Movie = {"year": 2022}  # Error (missing required field title)
```

Reply:
I don't have a strong opinion either way.

For inline comments like these I tend to use all lowercase and no full stops at the end, so even `ok` and `error` would be fine with me.
I don't see a strong reason to emphasize the `ERROR` since each example has a matching comment -- it's not a warning that needs to be made more noticeable otherwise people might miss it.

OTOH I would rephrase it to
```suggestion
   m3: Movie = {"year": 2022}  # ERROR (title is required, but missing)
```
which follows the previous example more closely and is IMHO more readable (with whatever case you settle on for `ERROR`).

Reply:
@AlexWaygood what about if we go with @ezio-melotti 's suggestion and not ALLCAPSing Error, like this?

```suggestion
   m3: Movie = {"year": 2022}  # Error (title is required, but missing)
```

Reply:
> @AlexWaygood what about if we go with @ezio-melotti 's suggestion and not ALLCAPSing Error, like this?

Sounds good to me.

Reply:
> For inline comments like these I tend to use all lowercase and no full stops at the end, so even ok and error would be fine with me.

Same, I don't add a period at the end either for inline or single-line comments...but I notice you _do_ use period for titles (and commit summary lines), which slightly confuses me :)  (To note, I do start them with a capital letter, which follows [the examples in PEP 8](https://peps.python.org/pep-0008/#comments) and avoids looking (IMO) sloppy—though that latter point is personal aesthetic preference)

Reply:
FWIW, in this general document, I'd appreciate a subtle indication that this is a *typing* error -- not a SyntaxError or runtime exception. Perhaps `Wrong`, in whatever case you prefer.

Reply:
That's a great point—to be more specific, we could say something like `Type check error` or `Incompatible type`